src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
Part of the idea of arcade-machine is that it provides enhancements on top of what browsers provide, not displacing them. The change made in 0.5.0 to rely on tab indexes was pursuant to that goal, essentially to make arcade-machine more standards-compliant.

If the tabIndex isn't -1, that means Edge thinks it's focusable, and screen readers and accessibility software will trip over that. A better fix would be to discover why it is being set to a non-zero value, and manually set it to -1 if necessary.